### DISCUSSION TITLE
Someone can help me to rejoin zKsync Discord ?

### DISCUSSION THREAD

**Discussion by AnyathianJade** [zksync/credo#39] [General]
I just got banned for discord because of sending nude links. it were accidentially and automatically happened. Now my account have fixd that problem . i still can't not join zkSync server. please someone can help me to get back in DC ?

**Comment by arnolwijaya77** [comment]
Fill out this form https://appeal.zksync.dev/

**Reply by AnyathianJade** [reply]
It's been 3 days bro, 😞

**Comment by AnyathianJade** [comment]
Thanks you so much!
i just filled it and submitted it. i hope that i could get back.
it super surprise because i didn't expected they will have appeal form.
Thanks in advance again bro.

**Reply by arnolwijaya77** [reply]
No worries, mods will review your request first. it may takes time.
